ludwig/visualize.py

```diff
@@ -24,18 +24,51 @@
 import numpy as np
 import pandas as pd
 import sklearn
+import h5py
 from scipy.stats import entropy
 from sklearn.calibration import calibration_curve
 from sklearn.metrics import brier_score_loss
 
 from ludwig.constants import *
 from ludwig.contrib import contrib_command, contrib_import
 from ludwig.utils import visualization_utils
-from ludwig.utils.data_utils import load_from_file, load_json
+from ludwig.utils.data_utils import load_from_file, load_json, load_array
 from ludwig.utils.print_utils import logging_level_registry
+from ludwig.features.feature_utils import retrieve_feature_hash
+from ludwig.utils.data_utils import (CSV_FORMATS, TSV_FORMATS, JSON_FORMATS,
+                                     JSONL_FORMATS, EXCEL_FORMATS,
+                                     PARQUET_FORMATS, PICKLE_FORMATS,
+                                     FWF_FORMATS,
+                                     FEATHER_FORMATS, HTML_FORMATS, ORC_FORMATS,
+                                     SAS_FORMATS, SPSS_FORMATS,
+                                     STATA_FORMATS, CACHEABLE_FORMATS,
+                                     read_csv, read_tsv, read_json, read_jsonl,
+                                     read_excel, read_parquet,
+                                     read_pickle, read_fwf, read_feather,
+                                     read_html, read_orc, read_sas,
+                                     read_spss, read_stata,
+                                     figure_data_format_dataset)
+from ludwig.utils.misc_utils import get_from_registry
 
 logger = logging.getLogger(__name__)
 
+cacheable_data_format_reader_registry = {
```

Comment:
this should live in data utils probaly.

Reply:
When I moved to the data_utils.py module, I noticed you already had a external reader registry built.  So I'm just reusing what you is already there instead of creating the `cacheable_data_format_reader_registry`.